### PR TITLE
Prepaid recharge docs

### PR DESCRIPTION
**Pull request by Arppp**
Preview docs

### CODE REVIEW

File: api-references/payments/prepaid-recharge.json [hunk 1]
```diff
@@ -0,0 +1,1210 @@
...
+                        "success": false,
+                        "trace_id": "10a5b1c5-9941-4551-b76a-35be8697c59d"
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "422": {
+              "description": "422",
```
Comment (aditya-67) [inline api-references/payments/prepaid-recharge.json:156]: Add a better description for 422 error

File: api-references/payments/prepaid-recharge.json [hunk 2]
```diff
@@ -0,0 +1,1210 @@
...
+                        "success": false,
+                        "trace_id": "3b0e103e-4fa8-403f-a6c9-b9dd62b82de7"
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "500": {
+              "description": "500",
```
Comment (aditya-67) [inline api-references/payments/prepaid-recharge.json:232]: Add a better description for 500

File: api-references/payments/prepaid-recharge.json [hunk 3]
```diff
@@ -0,0 +1,1210 @@
...
+                      },
+                      "description": "The params for details for the mobile number."
+                    }
+                  }
+                }
+              }
+            ]
+          }
+        },
+        "/mobile-prepaid-recharge/fetch_plans":{
```
Comment (aditya-67) [inline api-references/payments/prepaid-recharge.json:357]: Similar to above path comment, can this be `fetch-plans`? 

Let's drive consistency with our endpoints

File: api-references/payments/prepaid-recharge.json [hunk 4]
```diff
@@ -0,0 +1,1210 @@
+{
+    "info": {
```
Comment (aditya-67) [inline api-references/payments/prepaid-recharge.json:2]: Currently the base URL is using a default `example.com`. Can we add a host, scheme etc similar to [this](https://github.com/SetuHQ/docs/blob/main/api-references/payments/bbps.json#L2-L10)?

Screenshot 2024-10-23 at 10 26 24

**Review by aditya-67 — CHANGES_REQUESTED** [review]

File: api-references/payments/prepaid-recharge.json [hunk 5]
```diff
@@ -0,0 +1,1210 @@
...
+        "title": "Prepaid Mobile Recharge",
+        "x-logo": {
+          "url": "https://setu-customer-success.s3.ap-south-1.amazonaws.com/icons/setu-logo-dp-on-tp.svg",
+          "altText": "Prepaid Mobile Recharge"
+        },
+        "version": "1.0.0",
+        "description": "Process flow for prepaid recharge payments."
+      },
+      "paths": {
+        "/mobile-prepaid-recharge/fetch_operator_details": {
```
[the remark this answers is not in the document]
Comment (Arppp) [reply api-references/payments/prepaid-recharge.json:12]: mobile-prepaid-recharge is an identifier for route on Kong , do i make it /api/fetch-operator details or just 
prepaid.setu.co/fetch-operator-details

**Pull request merged**